datamanagement/source/Http/FoldersApi.gen.cs

```diff
@@ -903,7 +907,7 @@ private void SetHeader(string baseName, object value, HttpRequestMessage req)
             {
                 var queryParam = new Dictionary<string, object>();
                 SetQueryParameter("filter", filter, queryParam);
```

Comment:
@jmelhus we will also need it for the filter on the 'Search' method here.

[Documentation reference](https://aps.autodesk.com/en/docs/data/v2/developers_guide/filtering/)